tools/scoring/score/dimensions/entity_connection_identification.py

```diff
@@ -30,23 +33,30 @@ class EntityConnectionIdentification(Dimension):
   # rather than `translations` to do its calculations
   category = DimensionCategories.COMPLEX
 
-  def _isolate_connections(self, file: DeserializedFile):
+  def _isolate_connections(self, file: DeserializedFile) -> ConnectionsList:
     """Distill individual connections from each entity
     prior to inclusion in sets for global comparison."""
+    Connection = namedtuple('Connection', ['target', 'connection'])
+
     return [
-        tup for tup in (((cloud_device_id, connection)
-                         for connection in entity.connections)
-                        for cloud_device_id, entity in file.items()
-                        if entity.connections is not None) for tup in tup
+        Connection(entity.code, connection) for entity in file.values()
```

Comment:
can you please write this with the normal syntax, it is hard to follow.

Reply:
https://github.com/google/digitalbuildings/pull/561/commits/9eec8f21541d609a54f52c3bb3207c4fd4aa5832